src/agent/onefuzz-agent/src/tasks/fuzz/supervisor.rs

```diff
@@ -31,7 +31,7 @@ use tokio::{
     sync::Notify,
 };
 
-#[derive(Debug, Deserialize, Default)]
+#[derive(Debug, Deserialize)]
```

Comment:
Why did you remove the `Default`?  This was useful, considering how below you had to specify a suite of values are None.

Reply:
I had to remove it because the SyncDir No longer have an optional url. and Url does not implement Default. 
Unfortunately the `..Default::default()` syntax does not work even if you already specified the fields that require a value.